tools/mcsema_disass/ida/table.py

```diff
@@ -67,9 +67,16 @@ def __init__(self, inst, binary_is_pie):
     self.binary_is_pie = binary_is_pie
     self.candidate_target_eas = []
 
+    # Mask the computed target to have about the same number of bits as
+    # the jump instruction itself.
+    self.target_mask = 0xFFFFFFFFFFFFFFFF
```

Comment:
I see instances of missing blocks due to the target_mask. Do you think entry_size could be different in some cases?

Reply:
Yes. Entry size is usually 32- or 64-bit. I'm dealing with a case of a 32-bit entry size, with a 64-bit offset size, and so I'm trying to make things "work" with both apache and this windows binary. I'll have a new commit later today with another attempt.